back/app/services/xlsx_export/project_baskets_votes_generator.rb

```diff
@@ -17,15 +17,15 @@ def generate_project_baskets_votes_xlsx(project)
     private
 
     def generate_phase_baskets_votes_sheet(workbook, sheet_name, phase)
-      baskets = Basket.where(phase: phase).where.not(submitted_at: nil).includes([:user])
+      baskets = Basket.where(phase: phase).where.not(submitted_at: nil).includes([:user], [:baskets_ideas])
```

Comment:
I think you should be able to get rid of the square brackets here:
```suggestion
      baskets = Basket.where(phase: phase).where.not(submitted_at: nil).includes(:user, :baskets_ideas)
```